lib/scsi_defs.h

```diff
@@ -389,7 +389,7 @@ static_assert(sizeof(Read6Command) == 5);
 // SCSI Reference Manual Table 97
 // https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
 struct Read10Command {
-  uint8_t rd_protect : 3;  // read protect bit
+  uint8_t rdprotect : 3;   // read protect bit
```

Comment:
renaming because the field is RDPROTECT so the underscore didn't make sense in the first place.